plugins/exporter.koplugin/base.lua

```diff
@@ -140,4 +144,14 @@ function BaseExporter:toggleEnabled()
     end
 end
 
+--[[--
+Shares text with other apps
+]]
+function BaseExporter:shareText(text, title)
+    local msg_reason = _("Share") .. " " .. self.name
+    local msg_text = type(text) == "string" and text
+    local msg_title = type(title) == "string" and title
+    Device:doShareText(msg_text, msg_reason, msg_title, self.mimetype)
```

Comment:
It's no problem is some of these arguments are nil?

Reply:
Nope, the only device that `canShareText` is android.

And android ends calling https://github.com/koreader/android-luajit-launcher/blob/master/app/src/main/java/org/koreader/launcher/MainActivity.kt#L659, where everything other than text is nullable.